src/DynamoUtilities/PathHelper.cs

```diff
@@ -159,5 +159,12 @@ public static bool IsFileNameInValid(string fileName)
 
             return false;
         }
+
+        public static String GetSnapshotName(FileInfo fi)
```

Comment:
Would you add comments and description for new public Utility function?

Reply:
I would also be careful about design consistence of `pathHelper`, as you can tell all the other utility function all take a path as input. You may want to do the same and provide two APIs for your use, e.g. `GetSnapshotNameFromPath` and `getDirectoryNameFromPath`

Reply:
@QilongTang I changed the API to take the filePath as input parameter for the GetSnapshotNameFromPath function. I would not need GetDirectoryNameFromPath API, as we save the snapshot image to the same location as the graph location.